src/coreclr/jit/simd.h

```diff
@@ -1993,6 +1993,104 @@ SveMaskPattern EvaluateSimdMaskToPattern(var_types baseType, simdmask_t arg0)
         }
     }
 }
+
+template <typename TSimd, typename TBase>
```

Comment:
Same changes to this file also exist in #116852. I'll remove from one PR when the other is merged.